docs/In Depth Guides/Container-map-embed.md

```diff
@@ -0,0 +1,187 @@
+---
+stoplight-id: leasnylhe5su7
+tags: [map, embed]
+---
+
+# Container Map Embed Guide
+
+The Terminal49 Container Map allows you to embed real-time visualized container tracking on your website with just a few lines of code.
+
+### Prerequisites
+
+- Visit [getting started](https://developers.terminal49.com/docs/api/43898cdb7bad1-1-start-here#get-an-api-key) to get your developer API key and Terminal49 account.
+- Familiarity with Shipment and Container APIs.
+In the following examples we'll be passing a `containerId` and `shipmentId` variables to the embedded map.
+They relate to `id` attributes of the container and shipment objects that are returned by the API.
+See [Containers API](https://developers.terminal49.com/docs/api/f77d723c227c1-list-containers) and [Shipments API](https://developers.terminal49.com/docs/api/d31aeeda6fa44-list-shipments) for more information.
+
+### How do I embed the map on my website?
+
+Once you have the API Key, you can embed the map on your website.
+
+
+1. Copy and paste the code below and insert it on yor website.
+This will load and make the map code available through the global `window` object. 
+
+```html
+<link rel="stylesheet" href="https://map.terminal49.com/bundle.css" />
```

Comment:
Vercel still doesn't detect this domain, meaning this url is invalid.

@mattyturner could you help out with setting up a DNS so it redirects to Vercel's DNS?

![image](https://github.com/Terminal49/API/assets/15948633/2ea80030-bed4-4ce5-bce3-a52f1c170ff2)

cc @esbanarango